hudi-client/hudi-client-common/src/test/java/org/apache/hudi/config/TestHoodieWriteConfig.java

```diff
@@ -103,7 +103,7 @@ public void testSupportedTableWriteVersions() {
           Properties props = new Properties();
           props.setProperty(HoodieWriteConfig.WRITE_TABLE_VERSION.key(), String.valueOf(version.versionCode()));
           assertThrows(IllegalArgumentException.class, () -> HoodieWriteConfig.newBuilder()
-              .withPath("/tmp").withProperties(props).build());
+              .withPath("/tmp").withProperties(props).build().getWriteVersion());
```

Comment:
We landed this patch w/o waiting for full CI run
https://github.com/apache/hudi/commit/c7c3532477a611d94b8bb53af592bde2f151d764

thinking it just a warn msg we are removing. 
but apparently, one test is failing after this commit was merged. 
Fixing the test in this patch to un block master.